theme/templates/base.html

```diff
@@ -69,5 +67,22 @@
                 </ul>
             </div>
         </footer>
+        <!-- Google Analytics Reporter Script -->
```

Comment:
Move the reporter script inside the if statement and after the gtag/js file is loaded.